pontoon/base/static/js/tour.js

```diff
@@ -124,6 +128,6 @@ Sideshow.registerWizard({
   ]
 });
 
-if(Pontoon.state.project=='demo'){
+if (Pontoon.state.project == 'demo') {
```

Comment:
By default, it's better to always use `===` and `!==` in JS.

Reply:
Oh I will remember it. Thanks :D